WordPress/Classes/ViewRelated/System/Coordinators/MySitesCoordinator.swift

```diff
@@ -107,8 +107,9 @@ class MySitesCoordinator: NSObject {
         showRootViewController()
 
         if Feature.enabled(.newNavBarAppearance) {
-            blogListViewController.modalPresentationStyle = .pageSheet
-            mySiteViewController.present(blogListViewController, animated: true)
+            let navigationController = UINavigationController(rootViewController: blogListViewController)
```

Comment:
are these changes an active part of this PR or just added here for convenience? This code does not seem to be called in any of the scenarios. Just curious.

Reply:
I noticed that it was possible through this route for the blog list VC to be presented without a navigation controller (and so, no navigation bar). I figured I'd fix it while I was there :)